.eslintrc.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
   ],
   rules: {
     'prettier/prettier': ['error', {}, { usePrettierrc: true }],
-    'react/react-in-jsx-scope': 'off'
+    'react/react-in-jsx-scope': 'off',
+    'react/prop-types': 'off'
```

Comment:
Why disable the prop-types?

Reply:
Because we will have to define the prop-type for every single component. 
Should I remove this?

Reply:
We should always define them, right? So we know what the component is expecting, @invalidtoken?

Reply:
Yes, @ankushdharkar definitely.